yarn-project/sequencer-client/src/sequencer/index.ts

```diff
@@ -94,15 +94,15 @@ export class Sequencer {
    */
   protected async work() {
     try {
-      // Update state when the previous block has been synched
-      const prevBlockSynched = await this.isBlockSynched();
-      if (prevBlockSynched && this.state === SequencerState.PUBLISHING_BLOCK) {
-        this.log(`Block has been synched`);
+      // Update state when the previous block has been synced
```

Comment:
I believe we can start a flamewar around synced vs synched, only surpassable by tabs vs spaces

Reply:
Ill grab my pitchfork